src/module/display-home.js

```diff
@@ -1,6 +1,9 @@
 import showPopupComment from './show-popup-comment.js';
 import addLike from './add-like.js';
 import count from './count-items.js';
+import showlikes from './display-likes.js';
+
+const fetch = require('node-fetch');
```

Comment:
I would kindly recommend to delete this line, it's was add for testing but already not required.